jobs/kola-openstack.Jenkinsfile

```diff
@@ -105,7 +107,8 @@ lock(resource: "kola-openstack-${params.ARCH}") {
                     --name=${openstack_image_name} --arch=${params.ARCH}
                 """)
             }
-        
+
+
```

Comment:
only thing I would delete is remove this blank line addition

Reply:
Done